addons/stock/models/stock_move.py

```diff
@@ -1155,6 +1160,29 @@ def unlink(self):
         self.mapped('move_line_ids').unlink()
         return super(StockMove, self).unlink()
 
+    def split_mts(self):
```

Comment:
get a private methode (with underscore)  `def _split_mts(self):`
and add `self.ensure_one()`